src/features/users/user-controller.ts

```diff
@@ -379,6 +379,7 @@ class UserController {
             });
             emailSent = configurations.email.enabled;
         } catch (e) {
+            // TODO: Does it make sense for this to be rethrown?
```

Comment:
Yeah I thought about that too,
does it make sense for registration to fail if the email didn't go out

registration itself passed so I didn't rethrow the error